### PR TITLE
doc(data_management) : New Data Management page

### PR DESCRIPTION
**Pull request by jeromecambon**
- Created new page for data management

### CODE REVIEW

File: md/data-management.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Data Management in the UI Designer
+
+Discover how to explore [Business Data Model](define-business-data-model.md) (BDM) objects in the UI Designer, and use it to create user interfaces and variables.
+
+A **Data Model** tab is available from the Palette of the Page editor:
+
+![data_model_panel](images/data_model_panel.png)
+
+## Exploring the BDM
+From the Data Model tab, the graph icon allows you to visualize and explore the BDM.
```
Comment (benjaminParisel) [inline md/data-management.md:10]: Do you think it's a good idea to say here: "we use Voyager tool"? It's can be make some traffic on google if someone search voyager tool, we can guess we are in the result ?

File: md/data-management.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Data Management in the UI Designer
+
+Discover how to explore [Business Data Model](define-business-data-model.md) (BDM) objects in the UI Designer, and use it to create user interfaces and variables.
+
+A **Data Model** tab is available from the Palette of the Page editor:
+
+![data_model_panel](images/data_model_panel.png)
```
Comment (benjaminParisel) [inline md/data-management.md]: This screenshot is missing in your PR...

File: md/data-management.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## Create a user interface from a Business Object
+Drag and drop a Business Object in the whiteboard.
+A configuration wizard appears: provide information on how to retrieve elements of this Business Object.
+You can find information on how to use this wizard from the [Business Data Variables](variables.md) page.
+
+A user interface is generated. It follows the master/details pattern: a Table widget is displayed with all the object instances, and the details are shown when a line is selected.
+The details section map each Business Object attribute to a corresponding widget (for instance, an Input widget for an attribute of type String).
+A set of variables are generated as well, to allow the user interface to operate.
+Note that all the widgets are in read-only mode since the purpose here is to visualize data.
+You can try it with the Preview, and modify it as needed.
```
Comment (benjaminParisel) [inline md/data-management.md:43]: Can you add a disclaimer or something like that to inform user about pagination with datatable isn't working. 

And when we are written a how-to paginate your result...put the link here too

File: md/data-management.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ - Generate a user interface,
+ - Create a Business data variable and create yourself your user interface
+
+## Create a user interface from a Business Object
+Drag and drop a Business Object in the whiteboard.
+A configuration wizard appears: provide information on how to retrieve elements of this Business Object.
+You can find information on how to use this wizard from the [Business Data Variables](variables.md) page.
+
+A user interface is generated. It follows the master/details pattern: a Table widget is displayed with all the object instances, and the details are shown when a line is selected.
+The details section map each Business Object attribute to a corresponding widget (for instance, an Input widget for an attribute of type String).
```
Comment (benjaminParisel) [inline md/data-management.md:33]: If you want another widget for an attribute, you can use [switch widget feature](link).

**Review by benjaminParisel — CHANGES_REQUESTED** [review]
Small changed are needed, but nice work

File: md/data-management.md (hunk 2 above)
Comment (benjaminParisel) [reply md/data-management.md]: And call it with uid prefix can be better

File: md/data-management.md (hunk 1 above)
Comment (jeromecambon) [reply md/data-management.md:10]: Not sure we want to mention it, as discussed we may change the tool. Documenting it is a kind of commitment with it...

File: md/data-management.md (hunk 2 above)
Comment (jeromecambon) [reply md/data-management.md]: It was not a very good idea, but I uploaded it directly from github. Here it is:
data_model_panel

File: md/data-management.md (hunk 3 above)
Comment (jeromecambon) [reply md/data-management.md:43]: ok

File: md/data-management.md (hunk 4 above)
Comment (jeromecambon) [reply md/data-management.md:33]: ok

File: md/data-management.md (hunk 2 above)
Comment (benjaminParisel) [reply md/data-management.md]: Yes not a very idea. It's better to upload a new image in referenced PR ...

**Comment by florentsarat** [comment]
Hey,
We need to finish this page for the 7.11. Key information are available to the user but I don't it really highlights this cool feature...
Adding the "voyager" buzz word is good idea. We are already committed for the 7.11 so might as well put in the doc. If we change tech used, we will just update the doc.

Adding a few screenshots would help (it is a very graphical feature). The idea is that anyone skimming thru page will understand the feature

**Comment by jeromecambon** [comment]
> Hey,
> We need to finish this page for the 7.11. Key information are available to the user but I don't it really highlights this cool feature...
> Adding the "voyager" buzz word is good idea. We are already committed for the 7.11 so might as well put in the doc. If we change tech used, we will just update the doc.
> 
> Adding a few screenshots would help (it is a very graphical feature). The idea is that anyone skimming thru page will understand the feature

**Review by florentsarat — APPROVED** [review]

**Review by benjaminParisel — APPROVED** [review]

**Pull request merged**